test/commands/test-cli-helper/test-cli-errors.js

```diff
@@ -59,7 +59,9 @@ function errorIfStdRest(stds) {
   }
 }
 
-function errorIfBadCommand({ commandClass, commandArgs, commandPlan }) {
+function errorIfBadCommand({
```

Comment:
the diff in this file is to support the 'additionnalStep' keyword, which adds a step at the end of the plan.